lib/pat.c

```diff
@@ -1215,6 +1215,16 @@ pat_get(grn_ctx *ctx, grn_pat *pat, grn_id id)
   return grn_io_array_at(ctx, pat->io, SEGMENT_PAT, id, &flags);
 }
 
+static inline pat_node_common *
+_pat_get(grn_ctx *ctx, grn_pat *pat, grn_id id)
```

Comment:
Should we use the same prefix `pat_node`?
```suggestion
_pat_node_get(grn_ctx *ctx, grn_pat *pat, grn_id id)
```

Reply:
I've fixed them.